tests/proxy-test.yml

```diff
@@ -0,0 +1,47 @@
+config:
+  target: "http://localhost:3000"
+  http:
+    timeout: 10
+  phases:
+    - duration: 60
+      arrivalRate: 50
+      rampTo: 100
+    - duration: 60
+      arrivalRate: 100
+      rampTo: 200
+    - duration: 60
+      arrivalRate: 75
+      rampTo: 100
+    - duration: 300
+      arrivalRate: 100
+      rampTo: 200
+  processor: "./processor.js"
+
+scenarios:
+  - name: "View product page"
+    weight: 15
+    flow:
+      - function: "generateProductId"
+      - get:
+          url: "/{{ productId }}/"
+  - name: "View multiple product pages"
+    weight: 15
+    flow:
+      - function: "generateProductId"
+      - loop:
+        - get:
+            url: "/{{ productId }}/"
```

Comment:
Good use of variables, but from what I can tell this url only fetches html. I would recommend including a test that gets the json for a review via the proxy as well

Reply:
Oh nice catch! Yes, I forgot that this doesn't actually make the additional requests that the html would initiate. Good call.